test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```
@@ -116,7 +116,10 @@
     <None Update="Managment\Payloads\ExpectedHttpExtensionsPayload.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Update="Microsoft.Azure.WebJobs.Script.WebHost.deps.json">
+    <None Update="DepsFiles\net8.0\Microsoft.Azure.WebJobs.Script.WebHost.deps.json">
```

Could simplify to 
```suggestion
    <None Update="DepsFiles\*\Microsoft.Azure.WebJobs.Script.WebHost.deps.json" CopyToOutputDirectory="Always" />
```

And then remove the other lines.